src/tests/jobtest.c

```diff
@@ -30,26 +30,130 @@
 
 #include <stdio.h>
 #include <string.h>
+#include <strings.h>
 #include <daos.h>
+#include <daos_mgmt.h>
+#include <daos/common.h>
+#include <daos/pool.h>
+
+static char *progname;
+
+void print_usage()
```

Comment:
(style)  Bad function definition - void print_usage() should probably be void print_usage(void)